site/sitemap.xml

```diff
@@ -139,713 +139,721 @@
 </url>
 <!-- Auto-generate sitemap for rest of site content -->
 <url>
```

Comment:
Oops, did the sitemap get published last time with the localhost:4000 URLs? that's an error. Good change! this one is easy to miss and GIthub collapses this big diff.

Reply:
Yeah, I realized it. It was introduced unintentionally in the last one.

Reply:
Oh, thanks!